packages/fluentui/react-bindings/src/keyboardNavigationState/useKeyboardNavigationStateRef.ts

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { createKeyborg, disposeKeyborg, Keyborg } from 'keyborg';
+import { KeyboardNavigationState, KeyboardNavigationStateChangeCallback } from './types';
+
+class KeyboardNavigationStateImpl implements KeyboardNavigationState {
```

Comment:
You can implement this entire class with hooks and it would be more consistent with the rest of the codebase